zombienet_tests/misc/0002-upgrade-node.zndsl

```diff
@@ -2,8 +2,8 @@ Description: Smoke / Upgrade Node
 Network: ./0002-upgrade-node.toml
 Creds: config
 
-alice: parachain 2000 block height is at least 10 within 200 seconds
-bob: parachain 2001 block height is at least 10 within 200 seconds
+charlie: parachain 2000 block height is at least 10 within 200 seconds
```

Comment:
Why not query alice and bob?

Reply:
Since #6987, `cli args` changed (breaking change) and `ws-port` doesn't exist anymore. This test is using `latest` released version to start some nodes and then download the binary built in the `pr` , stop the process and replace the binary. To work around this, I changed zombienet to allow set the `cli args` version on the config and for this test I take the approach of set `cli args version 1` to alice/bob so they start with the latest version but they no expose a `ws` endpoint externally, **but** we update the binary we can access again using `ws` since the new binary have only one flag `rpc-port` and we use alice/bob to assert on the parachains progress.

Also, we can rollback to use alice/bob when this `cli args` changes were released.

Thanks!!